libs/accounts/email-renderer/src/renderer/email-helpers.ts

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import moment from 'moment-timezone';
+import safeUserAgent from 'fxa-shared/lib/user-agent';
+import { determineLocale } from '@fxa/shared/l10n';
+
+const DEFAULT_LOCALE = 'en';
+const DEFAULT_TIMEZONE = 'Etc/UTC';
+
+/**
+ * Email template helpers for formatting user agent, location, time, and other data
+ */
+export class EmailHelpers {
```

Comment:
This class has no members or internal state. So basically it has no reason to be class. In this case I think you should make these methods static,
or just have them as exported stand alone functions. Basically they don't benefit from object instantation.